deploy/Makefile

```diff
@@ -6,7 +6,7 @@
 # declare all targets in this variable
 ROOT_DIR:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 ALL_TARGETS:=deploy clean
-LOCALHOST_INVENTORY:=$(shell grep 'localhost ansible_connection=local' $(ROOT_DIR)/inventory)
+LOCALHOST_INVENTORY:=$(shell grep 'localhost ansible_connection=local' $(ROOT_DIR)/inventory > /dev/null 2>&1; echo $$?)
 ASK_BECOME_PASS:=""
```

Comment:
Remove this line, otherwise it's expanded as a double quoted empty string in the final command line:
https://github.com/intel/ccc-linux-guest-hardening/runs/7083169945?check_suite_focus=true#step:9:20

Reply:
thanks!

Reply:
You just needed to remove the line that I pointed: :)
`ASK_BECOME_PASS:=""`